templates/base.html

```diff
@@ -66,9 +66,11 @@
 		</main>
 		{% endblock %}
 		{% block footer %}
-			{% if crate::utils::sfw_only() %}
-			<footer><div id="sfw-only">This instance of Libreddit is SFW-only.</div></footer>
-			{% endif %}
+			<footer>
+				<div class="info-button">
+					<a href="/info">View instance information</a>
```

Comment:
At least Firefox 108.0.2, this does not appear centered. I assume you want this to be centered?

Reply:
@spikecodes changed this in abd24822 - I'm not sure whether this was intended. It was initially centered but maybe end-justified makes it less prominent on purpose. I'm not opposed to it either way, so it's up to you and Spike.

Reply:
Yeah, the right-alignment was intentional. My thinking is that having a centered info button right below the centered Next/Previous buttons (in page feeds) is too much of a conflict in design hierarchy.

Reply:
Gotcha, closing as resolved

Reply:
I'm not sure I like this. To me, it looks like a CSS/HTML mistake rather than something consistent with Libreddit's design rhetoric. I'll see if I can come up with something that addresses @spikecodes' concern.

This issue otherwise, this is ready to go in.